grip-mode.el

```diff
@@ -266,7 +266,8 @@ Use default browser unless `xwidget' is available."
     (add-hook 'kill-emacs-hook #'grip-stop-preview nil t)
     (cond ((derived-mode-p 'org-mode)
            (grip--preview-org))
-          ((derived-mode-p 'markdown-mode)
+          ((or (derived-mode-p 'markdown-mode)
```

Comment:
I think it's better as below. Your opion?

```emacs-lisp
(derived-mode-p 'markdown-mode 'markdown-ts-mode)
```

Reply:
I didn't realize that you can pass multiple modes to `derived-mode-p` - I learned something new today, thanks!

But it looks like that's the old calling convention and instead it should be passed a list, so I have updated the PR.